herbert_core/classes/instrument_classes/instrument/detectors/@IX_det_abstractType/IX_det_abstractType.m

```diff
@@ -77,6 +77,46 @@
         % Array of random points
         X = rand (obj, varargin)
     end
+
+    methods (Static)
+        function varargout = tobyfit_defaults(val,parms)
+        % TOBYFIT_DEFAULTS store and make available defaults for detector types
+        % e.g. He3Tube, Tobyfit_Classic, slab... so that they can be used to
+        % initialise detectors, until this information is available in nxspe input
```

Comment:
Is this a factory method? If so, the name is confusing. Not sure what this has to do with `defaults`

Reply:
additional explanation has been added to the comments for this